src/test/java/de/tum/in/www1/artemis/domain/FeedbackArchitectureTest.java

```diff
@@ -0,0 +1,27 @@
+package de.tum.in.www1.artemis.domain;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.noClasses;
+
+import org.junit.jupiter.api.Test;
+
+import com.tngtech.archunit.lang.ArchRule;
+
+import de.tum.in.www1.artemis.AbstractArchitectureTest;
+
+class FeedbackArchitectureTest extends AbstractArchitectureTest {
```

Comment:
I like this refactoring. Can you also apply this to the ArchUnit tests in `AuthorizationTest`?

Reply:
Done in f3d4800292919e5db21d6383aa016f4e04e1941a